libsolidity/ast/ASTJsonConverter.h

```diff
@@ -161,6 +168,7 @@ class ASTJsonConverter: public ASTConstVisitor
 		m_jsonNodePtrs.pop();
 	}
 
+	bool m_inEvent = false;
```

Comment:
Please change to `bool m_inEvent = false; ///< whethere we are currently inside an event or not`